src/containers/TurnitinDisplay/components/ButtonCell.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AlertModal, ActionRow, Button } from '@edx/paragon';
+
+const openInNewTab = url => {
+  window.open(url, '_blank', 'noopener,noreferrer');
+};
+
+export const ButtonCell = ({ value }) => (
+  <Button onClick={ () => openInNewTab(value)}>View report</Button>
```

Comment:
I would use `Hyperlink` component from paragon, checking it out here: https://paragon-openedx.netlify.app/components/hyperlink/#with-blank-target

Reply:
Thanks! I just updated the PR with this change!